config/routes.rb

```diff
@@ -68,7 +68,10 @@
       get  'result'
     end
 
-    get 'ready', on: :collection
+    collection do
+      get 'ready'
+      get 'users'
```

Comment:
Why is it even a separate action, not a `/polls?user=id` parameter on an index action?